File: app/src/main/java/com/ui/helper/log/IsLog.java
package com.ui.helper.log;

import android.util.Log;

import com.ui.helper.BuildConfig;

public class IsLog {

    public IsLog(String className, String print) {

        if (BuildConfig.BUILD_TYPE.equals("debug")) {
            Log.e(className,print);
        } else {
//            Log.e(className,print);
        }
    }

}

File: app/src/main/java/com/ui/helper/log/IsLog.kt
package com.ui.helper.log

import android.util.Log
import com.ui.helper.constant.GlobalData

class IsLog(className: String?, print: String?) {
    init {
        if (GlobalData.isDebugging) {
            Log.e(className, print.toString())
        }
    }
}
